src/unix.rs

```diff
@@ -113,7 +112,7 @@ impl HttpUnixConnector {
 impl NetworkConnector for HttpUnixConnector {
     type Stream = HttpUnixStream;
 
-    fn connect(&self, host: &str, port: u16, scheme: &str) -> hyper::error::Result<HttpUnixStream> {
```

Comment:
OK. In that case, please change the names to `_host`, `_port` and `_scheme` instead of `_`, so I can at least see what's going on here in the future. :-) Thank you!